pkg/gc/components.go

```diff
@@ -0,0 +1,20 @@
+package gc
+
+import (
+	"time"
+
+	config_core "github.com/kumahq/kuma/pkg/config/core"
+	"github.com/kumahq/kuma/pkg/core"
+	"github.com/kumahq/kuma/pkg/core/runtime"
+)
+
+var (
+	gcLog = core.Log.WithName("garbage-collector")
+)
+
+func Setup(rt runtime.Runtime) error {
+	if rt.Config().Environment != config_core.UniversalEnvironment {
+		return nil
+	}
+	return rt.Add(NewCollector(rt.ResourceManager(), 2*time.Second, rt.Config().Runtime.Universal.DataplaneCleanupAge))
```

Comment:
interval of 2 seconds for a process that fetches all dataplane insights (one of the biggest object in our system) using non-cached Resource Manager is overkill. Make it 1 minute, user won't care if the dataplane is removed after 3 days or after 3 days 1 minute